java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/TableOfflineIT.java

```diff
@@ -0,0 +1,125 @@
+
+@SystemTest
+public class TableOfflineIT {
+    @TempDir
+    private Path tempDir;
+    private PartitionTree expectedPartitions;
+    private FileReferenceFactory fileFactory;
+
+    @BeforeEach
+    void setUp(SleeperSystemTest sleeper, AfterTestReports reporting, AfterTestPurgeQueues purgeQueues) {
+        sleeper.connectToInstance(MAIN);
+        reporting.reportIfTestFailed(SystemTestReports.SystemTestBuilder::compactionTasksAndJobs);
+        purgeQueues.purgeIfTestFailed(COMPACTION_JOB_QUEUE_URL);
+        expectedPartitions = partitionsBuilder(sleeper).singlePartition("root").buildTree();
+        fileFactory = FileReferenceFactory.from(expectedPartitions);
+    }
+
+    @Test
+    void shouldNotSplitPartitionsIfTableIsOffline(SleeperSystemTest sleeper) {
+        // Given
+        sleeper.updateTableProperties(Map.of(
+                PARTITION_SPLIT_THRESHOLD, "20",
+                TABLE_ONLINE, "false"));
+        sleeper.setGeneratorOverrides(
+                overrideField(SystemTestSchema.ROW_KEY_FIELD_NAME,
+                        numberStringAndZeroPadTo(2).then(addPrefix("row-"))));
+        sleeper.ingest().direct(tempDir).numberedRecords(LongStream.range(0, 100));
+
+        // When
+        sleeper.partitioning().split();
+
+        // Then
+        assertThat(sleeper.directQuery().allRecordsInTable())
+                .containsExactlyInAnyOrderElementsOf(sleeper.generateNumberedRecords(LongStream.range(0, 100)));
+        Schema schema = sleeper.tableProperties().getSchema();
+        PartitionTree partitions = sleeper.partitioning().tree();
+        List<FileReference> activeFiles = sleeper.tableFiles().references();
+        assertThat(printPartitions(schema, partitions))
+                .isEqualTo(printPartitions(schema, expectedPartitions));
+        FileReferenceFactory fileReferenceFactory = FileReferenceFactory.from(expectedPartitions);
+        assertThat(printFiles(partitions, activeFiles))
+                .isEqualTo(printFiles(expectedPartitions, List.of(
+                        fileReferenceFactory.rootFile(100))));
+    }
+
+    @Test
+    void shouldNotCreateCompactionJobsIfTableIsOffline(SleeperSystemTest sleeper) {
+        // Given
+        sleeper.updateTableProperties(Map.of(
+                COMPACTION_FILES_BATCH_SIZE, "5",
+                TABLE_ONLINE, "false"));
+        // Files with records 9, 9, 9, 9, 10 (which match SizeRatioStrategy criteria)
+        RecordNumbers numbers = sleeper.scrambleNumberedRecords(LongStream.range(0, 46));
+        sleeper.ingest().direct(tempDir)
+                .numberedRecords(numbers.range(0, 9))
+                .numberedRecords(numbers.range(9, 18))
+                .numberedRecords(numbers.range(18, 27))
+                .numberedRecords(numbers.range(27, 36))
+                .numberedRecords(numbers.range(36, 46));
+
+        // When
+        sleeper.compaction().createJobs().invokeTasks(0);
```

Comment:
It doesn't seem like invokeTasks does anything here? This doesn't assert that no tasks will be created because it won't wait for long enough for the queue estimates to update.

We could assert that no jobs were created but I'm not sure how much value this adds since this is already tested elsewhere.